cmd/minikube/cmd/config/config.go

```diff
@@ -127,37 +140,44 @@ var settings = []Setting{
 		set:  SetBool,
 	},
 	{
-		name: config.ProfileName,
-		set:  SetString,
+		name:        config.ProfileName,
+		set:         SetString,
+		description: "The name of the minikube VM being used. (default \"minikube\")",
 	},
 	{
-		name: Bootstrapper,
-		set:  SetString,
+		name:        Bootstrapper,
+		description: "The name of the cluster bootstrapper that will set up the Kubernetes cluster. (default \"kubeadm\")",
+		set:         SetString,
 	},
 	{
-		name: "insecure-registry",
-		set:  SetString,
+		name:        "insecure-registry",
+		description: "Insecure Docker registries to pass to the Docker daemon.  The default service CIDR range will automatically be added.",
```

Comment:
Insecure image registries to pass to the container runtime engine (docker, containered, cri-o). The default service CIDR range will automatically be added.
...